src/button/Button.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react";
+import styled from "../../lib/styled";
+import t from "../../lib/theme";
+
+interface Props {
+  text?: string;
+}
+
+const StyledButton = styled.button({
+  color: "#fff",
+  background: `${t.colors.mojogreen}`,
+});
+
+const Button: React.SFC<Props> = ({ text, children }) => (
+  <StyledButton>{text || children}</StyledButton>
+);
+
+export default Button;
```

Comment:
This file seems to be a duplicate of `src/Button/Button.tsx`

Reply:
Yeah, i got some funky stuff happening after changing the case on the `Button|button` folder, will fix